src/main/java/org/kitteh/irc/client/library/feature/sts/StsClientState.java

```diff
@@ -0,0 +1,12 @@
+package org.kitteh.irc.client.library.feature.sts;
+
+/**
+ * Enum to keep track of the STS implementation's state machine.
+ */
+public enum StsClientState {
```

Comment:
One day KICL will be ported to C# and we will have [gloriously capitalised](https://msdn.microsoft.com/en-us/library/ms229043(v=vs.110).aspx#Capitalization%20Rules%20for%20Identifiers) class, enum and interface names. We will prefix our interfaces with `I`, stop shouting in our enums and throw away the getters and setters of yesterday in favour of properties! JavaDoc blocks will be shoved aside in favour of beautiful, concise XML based documentation and our fields and methods will once again be restored to their glory with PascalCase names.